README.md

```diff
@@ -0,0 +1,215 @@
+# API Advisor
+
+```json
+"baseURL": "https://csgo-dashboard-rank.herokuapp.com"
+```
+---
+## Summary
+- [Players](#players)
+  - [Register players](#register-players)
+- [Matches](#matches)
+  - [Register matches](#register-matches)
+- [Ranks](#ranks)
+  - [Kills](#rank-kills)
+    - [Display all kills sorted by higher to lower](#rank-kills-sorted)  
+
+---
+## <center>**Players** <a name="players"></a></center>
+---
+
+### Register players <a name="register-players"></a>
+
+```
+POST - /api/players/
+```
+Register a player into the database
+
+**Body:**
+```json
+{
+	"url": "https://csgostats.gg/player/76561198062653475"
+}
+```
+Response:
+```
+HTTP: 200 - OK
+```
+```json
+{
```

Comment:
probably no big deal, but could we do 2 spacings, i think these jsons are really spaced, something like this:
```json
{
  "playerId": "850fde80-8231-452t-b39c-f9b23695385a",
  "player": "SK|Blavold",
  "imageUrl": "https://avatars.akamai.steamstatic.com/e66064d13f75fe43da64fce740ac29a217f9fd6a_full.jpg",
  "platformCredentials": [
    {
      "platformName": "CSGOSTATS",
      "platformPlayerId": "76561198062653475"
    }
  ]
}
```

Reply:
done